toolkit/scripts/containerized-build/resources/welcome.txt

```diff
@@ -21,10 +21,10 @@
                                                         create new folder under /usr/src/mariner/SPECS
                                                         with .spec (and other files)
 
-*     Changes to /usr/src/mariner/SPECS will be available on host machine at <CBL-Mariner>/SPECS
+*     Changes to /usr/src/mariner/SPECS will be available on host machine at <Azure Linux>/SPECS
```

Comment:
I believe this should read "azurelinux"/SPECS.  Not "Azure Linux"/SPECS

Same with the subsequent change

Reply:
Fixed it